src/fixup.js

```diff
@@ -194,7 +198,18 @@
           button.innerText = (isOpen) ? '\u25BE collapse' : '\u25B4 expand';
         }
       }
+
       document.body.appendChild(node);
+      button.focus();
+      window.onkeydown = function (event) {
+        if (event.keyCode == 27 && !node.classList.contains("outdated-collapsed")) button.click();
+      }
+      document.addEventListener("focus", function(event) {
+        if (!node.classList.contains("outdated-collapsed") && !node.contains(event.target)) {
+          event.stopPropagation();
+          node.focus();
+        }
+      }, true);
```

Comment:
nit: why`true` here? If you really want to use capture, you should leave a comment why.

Reply:
addressed